images/installer/Dockerfile.upi.ci

```diff
@@ -0,0 +1,35 @@
+# This Dockerfile is a used by CI to test UPI platforms for OpenShift Installer
+# It builds an image containing binaries like jq, terraform, awscli, oc, etc. to allow bringing up UPI infrastructure.
+# It also contains the `upi` directory that contains various terraform and cloud formation templates that are used to create infrastructure resources.
+
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+RUN hack/build.sh
+
+FROM centos:7
+COPY --from=registry.svc.ci.openshift.org/ocp/4.0:cli /usr/bin/oc /bin/oc
```

Comment:
All of this is probably getting clobbered in the release repo, but it feels strange to me to reach into openshift/release for `golang-1.10` and then into `ocp` for the `cli`.  Can we pull `cli` from somewhere more origin-y?

Reply:
done  91cd579 to 512f93d